NYPLAudiobookToolkit/UI/AudiobookPlayerViewController.swift

```diff
@@ -82,7 +82,6 @@ let SkipTimeInterval: Double = 15
         self.activityIndicator.color = NYPLColor.disabledFieldTextColor
         self.audiobookManager.audiobook.player.registerDelegate(self)
         self.audiobookManager.networkService.registerDelegate(self)
-        self.audiobookManager.networkService.fetch()
```

Comment:
Removing this because we do not want to fetch whenever the player is presented, that would cause the download happen outside of the `NYPLAudiobookDownloader` class and unable to log errors nor display the download progress properly.